Octokit.Reactive/Clients/IObservableRepositoryCommitsClients.cs

```diff
@@ -40,11 +40,28 @@ public interface IObservableRepositoryCommitsClient
         /// </summary>
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
+        /// <param name="options">Options for changing the API response</param>
```

Comment:
As long as you changed XML documentation, could you add adjust <returns> tag here and on other places?
You could use this pattern:

`/// <returns>The <see cref="EmailAddress"/>es for the authenticated user.</returns>`